src/AnnotationReader/Reader.php

```diff
@@ -24,7 +24,7 @@ public function getAnnotations($item): array
     /**
      * @param \ReflectionClass|\ReflectionMethod|\ReflectionProperty $item
      *
-     * @throws \SofaScore\Purgatory\AnnotationReader\ReaderException
+     * @throws \Sofascore\PurgatoryBundle\AnnotationReader\ReaderException
```

Comment:
@HypeMC FQCN is redundant here.

Reply:
Done, but in the other PR https://github.com/sofascore/purgatory/pull/55/commits/e6798573286be9591f20537ca048c50b3b8c2926